documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,38 @@
+# TWAP
+
+## Overview
+
+TWAP strategy expands upon Simple Trade strategy. By the end of this part, you should be able to : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
+
+## Config
+
+Modifications for the following fields have to be made to the `config_map` file :
+
+* `time_delay` : Change the question to ask for number of seconds to delay each individual order. (e.g. How many seconds do you want to wait between each individual order?)
+
+* `num_individual_orders` : This is is a new field added to the config map. It should ask for number individual orders that an order should be split up into. (e.g.Into how many individual orders do you want to split this order?)
+
+## Strategy file
+
+`self._quantity_remaining` : Indicates quantity of order left to placed as individual orders
```

Comment:
This section needs to be re-organized.

While giving information in point form is useful for scanning... you also need to give the reader context in which to understand them.

You should always start from talking about the overall picture / bird-eye view first - in which the flow chart diagram is useful - before you drill down into the specifics like attributes and functions. If you read into other dev documentation pages (e.g. https://docs.hummingbot.io/developers/strategies/arbitrage/)... we always start with some high-level introductory text, followed by diagram, followed by more detailed description, before we refer to the code.